config/application.php

```diff
@@ -170,7 +170,7 @@
 Config::define('SCRIPT_DEBUG', false);
 ini_set('display_errors', '0');
 // Additional logging for the authentication mu-plugin.
-Config::define('MOJ_AUTH_DEBUG', true);
+Config::define('MOJ_AUTH_DEBUG', env('MOJ_AUTH_DEBUG') ?? false);
```

Comment:
Hi @EarthlingDavey 
Just an observation; why are we setting a second global variable? 

This...
```php
echo env('MOJ_AUTH_DEBUG');
echo MOJ_AUTH_DEBUG;
```
It will yield the same value.

Reply:
Oh... I see. The constant can be false as well as true. Is that right?

Reply:
Ah good spot! Considering I'm only checking if it's true in the codebase, I don't think I need to set the default value as false.